src/components/destination_list.js

```diff
@@ -8,26 +8,46 @@ import NoItems from "./items/no_items";
 import SelectedItem from "./items/selected_item";
 import SelectionStatus from "./selection_status/selection_status";
 import { groupItems } from "./item_grouping_util";
+import Search from "./search/search";
 
 const DestinationList = ({
   selectionStatusRenderer,
   selectedIds,
   clearAll,
   messages,
-  selectedItems,
   itemHeight,
   height,
   unselectItems,
   selectedItemRenderer,
   noItemsRenderer,
-  withGrouping
+  withGrouping,
+  searchIcon,
+  selectedItems,
+  showDestinationSearch,
+  filterDestinationItems,
+  searchDestinationValue,
+  filteredDestinationItems
 }) => {
   const SelectionStatusRenderer = selectionStatusRenderer;
+  const filteredSelectedItems =
+    showDestinationSearch && filteredDestinationItems.length > 0
```

Comment:
`filteredDestinationItems.length > 0`
isn't this preventing from empty results showing?
If there is no match we need to show nothing